.github/workflows/build_dataops.yml

```diff
@@ -0,0 +1,48 @@
+name: Build jupyterlab-dataops Image
+
+on:
+  pull_request:
+    paths:
+      - 'jupyterlab-dataops/**'
```

Comment:
You need to create a branch on the official repo to make the build work (otherwise GCR password is not provided)